packages/runtime/src/modules/AttributeListenerModule.ts

```diff
@@ -19,20 +21,68 @@ export class AttributeListenerModule extends RuntimeModule {
         const createdAttribute = event.data;
         if (createdAttribute.content["@type"] === "IdentityAttribute" && createdAttribute.shareInfo) return;
         if (createdAttribute.content["@type"] === "RelationshipAttribute" && createdAttribute.content.confidentiality === RelationshipAttributeConfidentiality.Private) return;
+        if (await AttributeListenerModule.relationshipAttributeOfInactiveRelationship(services, createdAttribute)) return;
 
         const getAttributeListenersResult = await services.consumptionServices.attributeListeners.getAttributeListeners();
         if (getAttributeListenersResult.isError) {
-            this.logger.error("Could not get attribute listeners", getAttributeListenersResult.error);
+            this.logger.error("Could not get LocalAttributeListeners", getAttributeListenersResult.error);
```

Comment:
it was better before, the modules consume the runtime and do not know anything about local stuff.

Reply:
I have no problem with undoing my changes. Nevertheless, I do not understand exactly why it is not more precise to speak of LocalAttributeListeners from the naming of the return types of the functions called here. An "attribute listener" would then not be a data object type, but just a word from normal usage?

Reply:
It could be misleading to talk about Local stuff as local is just an internal wording we don't really expose anywhere.